handbook/product/design/design_process.md

```diff
@@ -98,7 +98,12 @@ While asynchronous communication is a core attribute of remote work, key moments
 
 - Visual design should utilize the Sourcegraph’s Figma based component system
 - Dark compositions will be created for each major design
-- If new components are required, components will be created in the file itself, not the component system
+- If new components are required, the following process should be followed
+    - Design component in the Figma file which requires it
```

Comment:
@rrhyne I would add your requirements that you've mentioned in Figma, here:

1. Proposal with visual examples of Storybook, in product usage and suggested change
2. Sourcegraph search showing affected areas of the product
3. Visual examples of how the proposal affects those areas
4. Github issue for development when the review is complete